app/__init__.py

```diff
@@ -1,16 +1,19 @@
 from flask import Flask
 from flask_restful import Api
 from flask_login import LoginManager
+from flask_bootstrap import Bootstrap
 
 import os
 
 api = Api()
+
 login_manager = LoginManager()
 login_manager.login_view = "login"
 
 
 def create_app(name=__name__):
     app = Flask(name)
+    bootstrap = Bootstrap(app)  # noqa: F841
```

Comment:
Create the bootstrap object outside of the application factory. You should have something like
```python
bootstrap = Bootstrap()
...
def create_app(name=__name__):
    ...    
    bootstrap.init_app(app)
```